packages/caliper-fabric/lib/ConnectorConfiguration.js

```diff
@@ -86,6 +86,22 @@ class ConnectorConfiguration {
 
         return null;
     }
+    /**
+     * return array of contract definitions for specific channel
+     * @param {string} channelName channel name wanted
+     * @returns {Array} all of contract definitions for channel
+    */
+    getContractDefinitionsForChannelName(channelName) {
+        const channelList = this.adapterConfiguration.channels;
+
+        if (channelList && Array.isArray(channelList)){
+            const channelWanted = channelList.filter(channelContracts => channelContracts.channelName === channelName);
+            const contractDefinitions = channelWanted[0].contracts;
```

Comment:
Should test here what is in contracts, eg
```
if (contractDefinitions && 
    Array.isArray(contractDefinitions) {
   return contractDefinitions;
}
```
Then add a test to see that you get an empty array is the contracts property hasn't been specified